gitbootcamp.rst

```diff
@@ -205,51 +201,48 @@ Commit the files:
 
 .. code-block:: bash
 
-   git commit -m '<message>'
-
+   git commit -m "<message>"
```

Comment:
The quotation marks aren't needed here, since it is already using <>.
```suggestion
   git commit -m <message>
```

Reply:
Okay.